tests/generators/kzg_4844/main.py

```diff
@@ -27,11 +27,11 @@ def expect_exception(func, *args):
 
```

Comment:
Hmm, do we also need to change the specification in `tests/formats/kzg`? If we are in a hurry, we can do it on a subsequent PR?

Reply:
Done in a7f865972